subcommand/lifecycle-sidecar/command.go

```diff
@@ -47,12 +48,12 @@ func (c *Command) init() {
 	flags.Merge(c.flagSet, c.http.Flags())
 	c.help = flags.Usage(help, c.flagSet)
 
-	// Wait on an interrupt to exit. This channel must be initialized before
+	// Wait on an interrupt or terminate to exit. This channel must be initialized before
 	// Run() is called so that there are no race conditions where the channel
 	// is not defined.
 	if c.sigCh == nil {
 		c.sigCh = make(chan os.Signal, 1)
-		signal.Notify(c.sigCh, os.Interrupt)
+		signal.Notify(c.sigCh, os.Interrupt, syscall.SIGTERM)
```

Comment:
Update `os.Interrupt` to `syscall.SIGINT` for consistency.
```suggestion
		signal.Notify(c.sigCh, syscall.SIGINT, syscall.SIGTERM)
```